packages/vscode-ui5-language-assistant/test/unit/settings-config-spec.ts

```diff
@@ -0,0 +1,45 @@
+import { expect } from "chai";
+import { readJsonSync } from "fs-extra";
+import { forEach, set } from "lodash";
+import { getDefaultSettings } from "@ui5-language-assistant/settings";
+
+describe("settings configuration properties", () => {
+  let packageJsonSettings: Record<string, Setting>;
+  before(() => {
+    // Get the settings from the package.json
+    const packageJsonPath = require.resolve(
+      "vscode-ui5-language-assistant/package.json"
+    );
+    const packageJsonContent = readJsonSync(packageJsonPath);
+    packageJsonSettings =
+      packageJsonContent.contributes.configuration.properties;
+  });
+
+  it("default setting values in package.json have the correct type", () => {
```

Comment:
not important enough to upgrade, but I would dynamically create tests (`it` calls)
one for each property so that a possible failure would have a better error message, e.g:

> property `foo` has default value mismatch between pkg.json and runtime defaults

Reply:
To create an `it` for every setting I'll have to read the `package.json` outside the `before`

Reply:
I did change the error messages